wandb/__init__.py

```diff
@@ -1073,6 +1073,8 @@ def init(job_type=None, dir=None, config=None, project=None, entity=None, reinit
 
     try:
         signal.signal(signal.SIGQUIT, _debugger)
+    except ValueError:
+        pass
     except AttributeError:
```

Comment:
We can actually do `except (ValueError, AttributeError):` here.  I can update in the next release or happy to take your PR if you make the change.  Thanks for this!